app/src/main/java/com/example/poste/activities/DashboardActivity.java

```diff
@@ -39,21 +39,25 @@
  * The DashboardActivity class adds functionality to the activity_dashboard.xml layout
  */
 public class DashboardActivity extends PActivity {
-    private User currentUser;
     private Button addButton;
     private Button searchButton;
-    private List<com.example.poste.models.Folder> userFolders;
     private RecyclerView folderRecyclerView;
     private FolderAdapter folderAdapter;
     private ImageView optionsView;
     private SwipeRefreshLayout swipeRefreshLayout;
     private final MyApiService apiService = RetrofitClient.getRetrofitInstance().create(MyApiService.class);
     private UpdateCallback updateCallback;
+    private TextView dashboardViewEmptyText;
     
     @Override
     protected void onRestart() {
         super.onRestart();
-        reloadDashboard();
+        updateData();
+    }
+
+    private void updateData() {
+        dashboardViewEmptyText.setVisibility(View.GONE);
```

Comment:
Sets the empty dashboard notification to invisible; the callback makes it visible if there are no folders (avoids the notification popping in and out)